src/main/java/racingCar/domain/Cars.java

```diff
@@ -0,0 +1,55 @@
+package racingCar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private static final String SPLIT_DELIMITER = ",";
+    public static final int RANDOM_LIMIT_VALUE = 10;
+
+    private List<Car> carList;
+
+    public Cars(String carNames){
+        carList = new ArrayList<>();
+        String[] carNameArray = splitCarNames(carNames);
+
+        for(int i=0; i<carNameArray.length; i++) {
+            this.carList.add(new Car(carNameArray[i]));
+        }
+    }
+
+    public Cars(int carCount){
+        carList = new ArrayList<>();
+        for(int i=0; i<carCount; i++){
+            this.carList.add(new Car(0));
+        }
+    }
+
+    public Cars(List<Car> carList){
+        this.carList = carList;
+    }
+
+    public List<Car> getCarList(){
+        return carList;
+    }
+
+    private String[] splitCarNames(String carNames){
+        return carNames.split(SPLIT_DELIMITER);
+    }
+
+    public List<Car> addCarMoving(){
```

Comment:
```suggestion
    public List<Car> addCarMoving(RandomNumber randomNumber) {
```

테스트 가능하게 하기위해 주입을 받는것도 고려해볼수 있겠네요~

Reply:
RandomNumber를 객체로 만들면 RandomNumber의 기본 생성자에서 인스턴스 변수를 랜덤 숫자로 초기화해 줘야 할 것 같은데, addCarMoving 메소드 인자로  RandomNumber를 받으면 for문 안에서 매번 랜덤 숫자를 생성해줄 방법이 있을까요?

현재처럼 메소드 인자는 받지 않고 아래와 같이 변경하는 방법밖에 안떠오르는데 RandomNumber를 주입받아서 하는 방법이 있을까요..😂
```
//addCarMoving 메소드
for(Car car : carList){
    car.moveCar(new RandomNumber());         
    carStatusList.add(car.carStatus(SEPARATE_DELIMITER, FORWARD_MARK));
}

//RandomNumber 생성자
public RandomNumber() {
    this.randomNumber = new Random().nextInt(RANDOM_LIMIT_VALUE);
}
```